ExampleSwift/ExampleSwift.xcodeproj/project.pbxproj

```diff
@@ -646,6 +604,25 @@
 			defaultConfigurationName = Release;
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
+		5864396F2B7D17680031140F /* XCRemoteSwiftPackageReference "attentive-ios-sdk" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/attentive-mobile/attentive-ios-sdk";
+			requirement = {
+				branch = "rsmith/support-swift";
```

Comment:
will adapt this to be main in a new PR with version specified once this is merged